File: platform/College-1/Litcoder/Week 1/Code/lab3.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        String str = input.nextLine();
        String[] arrStr = str.split(" ");
        int arrLen = arrStr.length;
        int sum = 0;
        if(arrLen>10) {
            System.out.print("Array size must be between 1 and 10");
            return;
        }
        try {
            for(int index=0;index<arrLen;index++) {
                int element = Integer.parseInt(arrStr[index];
                if(element<-10 || element>10) {
                    System.out.print("Array elements must be from -10 to 10");
                    return;
                }
                sum = sum + element;
            }
        } catch(NumberFormatException e) {
            System.out.print("Array elements must be integers");
            return;
        }
        System.out.println((sum==0)?"True":"False");
        System.out.print(arrLen);
        input.close();
    }
}

File: platform/College-1/Litcoder/Week 1/Code/lab3.py
def find_subarray(arr):
    arr = arr.split(' ')
    sum = 0
    arr_len = len(arr)
    if (arr_len > 10):
        print("Array size must be between 1 and 10",end='')
        return
    try:
        for index in range(arr_len):
            element = int(arr[index])
            if(element<-10 or element>10):
                print("Array elements must be from -10 to 10")
            sum = sum + element
    except ValueError:
        print("Array elements must be integers",end='')
        return
    print((sum==0))
    
    print(arr_len,end='')
find_subarray(input())
